src/docs/03-concepts/10-http-api.md

```diff
@@ -5,208 +5,2181 @@ permalink: /docs/concepts/http-api
 ---
 
 # Using HTTP API
-Cadence allows you to interact with server using HTTP API. What does it mean?  HTTP/JSON communication is a versatile way to interact with servers. For Cadence, it means you can expose a list of RPC methods to be invoked using HTTP protocol. This opens possibilities to interact with Cadence server using any programming language with HTTP support. This can be used to start or terminate workflows from your bash scripts, observe cluster status or invoke any other operation supported by Cadence RPC declaration. 
 
- 
+## Introduction
 
-## Setup 
+From version 1.2.1 onwards, Cadence has introduced HTTP API support, which allows you to interact with the Cadence server 
```

Comment:
maybe you want emphasize (bold) the phrase "version 1.2.1"?

Reply:
Done